repose-aggregator/components/filters/api-validator-filter/src/main/java/org/openrepose/filters/apivalidator/ApiValidatorHandler.java

```diff
@@ -153,9 +146,16 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
                 }
 
                 if (!isValid) {
-                    if (mbcsInvalidRequests != null) {
+                    if (Optional.ofNullable(metricsService).isPresent()) {
+                        MetricRegistry metricRegistry = metricsService.getRegistry();
                         for (String s : matchedRoles) {
-                            mbcsInvalidRequests.mark(s);
+                            metricRegistry.meter(
```

Comment:
Oops. I was being careless. This is a dynamic metric that were were passing a string to before, because we had those category classes.